### ISSUE TITLE
Multi label classification

### ISSUE THREAD

**Issue by FerreroJeremy** [FerreroJeremy/tadashi#5]
## Do multi-label classification instead of single-label classification

In `train` method, use softmax activation, instead of sigmoid, for the last layer of the neural network and in `classify` method, change the number of conserved labels from the prediction.

Morevover, in `Linker` don't forget  to allow the possibility to link multiple classes to a map.

*Remark*
We compile the model using binary cross-entropy rather than categorical cross-entropy.
This may seem counterintuitive for multi-label classification; however, the goal is to treat each output label as an independent Bernoulli distribution and we want to penalize each output node independently.

**Comment by FerreroJeremy** [comment]
**TODO:**
1. Replace the softmax activation at the end of the network with a sigmoid activation
2. Swap out categorical cross-entropy for binary cross-entropy for the loss function
